src/components/SelectStatus/SelectButtonBox/index.tsx

```diff
@@ -0,0 +1,42 @@
+'use client';
+
+import type { Dispatch, SetStateAction } from 'react';
+import React from 'react';
+
+import * as S from './style';
+
+import { MenteeSelectIcon, MentorSelectIcon } from '@/assets';
+
+interface Props {
+  role: 'mentee' | 'mentor' | null;
+  setRole: Dispatch<SetStateAction<'mentee' | 'mentor' | null>>;
+}
+
+const SelectButtonBox: React.FC<Props> = ({ role, setRole }) => {
+  const onChangeState = (currentState: 'mentee' | 'mentor') => {
+    setRole(currentState);
+  };
+
+  return (
+    <S.Container>
+      <S.SelectButton
+        isActive={role === 'mentee'}
+        onClick={() => onChangeState('mentee')}
```

Comment:
```suggestion
        onClick={() => setRole('mentee')}
        onClick={() => handleRoleSelectClick('mentee')}
```

워낙 간단한 동작이므로 바로 setRole을 사용해도 괜찮을 것 같습니다.

또는 이벤트 핸들러 네이밍을 조금 수정해도 좋을 것 같아요

Reply:
1c3819f024c6cd4704a2991bc9af0e2f9835c6ae

수정하였습니다!